Assets/Tests/Weaver/BitAttributeTests.cs

```diff
@@ -179,22 +179,22 @@ public void Vector3PackInvalid()
                "UnityEngine.Vector3 BitAttributeTests.Vector3PackInvalid.MyBehaviour::value7");
 
 #if UNITY_2021_2_OR_NEWER
-            HasError("Max must be above 0, max:(-1.00, 0.00, 0.00)",
+            HasError($"Max must be above 0, max:({-1.0:0.00}, {0.0:0.00}, {0.0:0.00})",
                "UnityEngine.Vector3 BitAttributeTests.Vector3PackInvalid.MyBehaviour::value8");
-            HasError("Max must be above 0, max:(1.00, -1.00, 0.00)",
+            HasError($"Max must be above 0, max:({1.0:0.00}, {-1.0:0.00}, {0.0:0.00})",
                "UnityEngine.Vector3 BitAttributeTests.Vector3PackInvalid.MyBehaviour::value9");
-            HasError("Max must be above 0, max:(1.00, 1.00, -1.00)",
+            HasError($"Max must be above 0, max:({1.0:0.00}, {1.0:0.00}, {-1.0:0.00})",
                "UnityEngine.Vector3 BitAttributeTests.Vector3PackInvalid.MyBehaviour::value10");
-            HasError("Max must be above 0, max:(-1.00, 0.00, 0.00)",
+            HasError($"Max must be above 0, max:({-1.0:0.00}, {0.0:0.00}, {0.0:0.00})",
                "UnityEngine.Vector3 BitAttributeTests.Vector3PackInvalid.MyBehaviour::value11");
 #else
-            HasError("Max must be above 0, max:(-1.0, 0.0, 0.0)",
+            HasError($"Max must be above 0, max:({-1.0:0.0}, {0.0:0.0}, {0.0:0.0})",
                "UnityEngine.Vector3 BitAttributeTests.Vector3PackInvalid.MyBehaviour::value8");
-            HasError("Max must be above 0, max:(1.0, -1.0, 0.0)",
+            HasError($"Max must be above 0, max:({1.0:0.0}, {-1.0:0.0}, {0.0:0.0})",
                "UnityEngine.Vector3 BitAttributeTests.Vector3PackInvalid.MyBehaviour::value9");
-            HasError("Max must be above 0, max:(1.0, 1.0, -1.0)",
+            HasError($"Max must be above 0, max:({1.0:0.0}, {1.0:0.0}, {-1.0:0.0})",
                "UnityEngine.Vector3 BitAttributeTests.Vector3PackInvalid.MyBehaviour::value10");
-            HasError("Max must be above 0, max:(-1.0, 0.0, 0.0)",
+            HasError($"Max must be above 0, max:({-1.0:0.0}, {0.0:0.0}, {0.0:0.0})",
```

Comment:
Can you add a small comment about why the string interpolation, just so we dont forget

Reply:
Done